bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/AnomalousCouplings-SMP/WWTolnulnu_01j_aTGC_lep_WWmass_4f_NLO_FXFX_UL/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus/WWTolnulnu_01j_aTGC_lep_WWmass-400to600_4f_NLO_FXFX_W_minus_proc_card.dat

```diff
@@ -0,0 +1,18 @@
+import model EWdim6NLO-no_cmass_cwwwcbcw
+
+define j = g u c d s u~ c~ d~ s~
+define p = g u c d s u~ c~ d~ s~
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+define V = w+ w-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+
+generate p p > ell+ vl w- $$ t t~ H QED=3 [QCD] @0
+add process p p > ell+ vl w- j $$ t t~ H QED=3 [QCD] @1
+#add process p p > ell- vl~ w+ $$ t t~ H QED=3 [QCD] @2
+#add process p p > ell- vl~ w+ j $$ t t~ H QED=3 [QCD] @3
```

Comment:
I still see unnecessary lines like
define V
and lines with #add process

Reply:
Right, I missed those. Now it is fixed